src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/action_page.php"
+      method="get"
+      data-qa="big"
+    >
+      <label for="search"></label>
+      <input
+        type="search"
+        id="search"
+        name="search"
+        data-qa="keypress"
+        class="search"
+        placeholder="Try “Los Angeles”"
+      >
+    </form>
+
+    <form
+      action="/action_page.php"
+      method="get"
```

Comment:
Consider using the POST method instead of GET for the search form if you are sending sensitive data or dealing with a large amount of data.